src/client/components/dashboard/dashboard_robot/model.ts

```diff
@@ -79,28 +78,27 @@ export class DashboardRobotModel {
 
   static of = memoize((robot: RobotModel): DashboardRobotModel => {
     return new DashboardRobotModel(robot, {
-      ballColor: '#ff9800',
+      ballColor: 0xff9800,
       ballCovariance: Matrix2.of(),
       ballPosition: Vector2.of(),
-      ballSightColor: '#4DB6AC',
+      ballSightColor: 0x4DB6AC,
```

Comment:
Seeing the raw number `5093036` in the console as a variable is going to suck, the bit-shifting needed below is already a bad sign. Can we go the other way and parse hex strings in the pixi renderer? `parseInt('#4DB6AC'.slice(1), 16)` reads much nicer.

Reply:
I thought about it, but it ends up being dangerous since before it was just a style which could have anything in it. Limiting it down to colour works better. Is there some typescript way that I can enforce it to be a string of the form `#[A-Za-z0-9]{6}`

Reply:
Just validate it at parse time, I think that should be fine and the least annoying.

```ts
function hexColorInt(hex: string): number {
  if (!hex.match(/^#[0-9a-f]{6}$/) {
    throw new Error(`Invalid color: ${hex}`)
  }
  return parseInt('#4DB6AC'.slice(1), 16)
}
```